schema.sql

```diff
@@ -5,5 +5,52 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL,
-    species VARCHAR(20)
-);
\ No newline at end of file
+    species_id INT,
+    owner_id INT
+);
+
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(300),
+    age INT
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(300)
+);
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals
```

Comment:
- [ ] Kindly add  a modification to  remove the species column entirely from the animal's table

### Instructions:
![image](https://github.com/devalibello/vet-clinic/assets/50990925/7194df90-ee45-4864-9298-b950270849b0)

Reply:
Hello @Shedrack-Sunday
The section of the code you highlighted adds a foreign key reference to the species_id column. The species column has already been removed from the animals table as indicated in the screenshot below.

![image](https://github.com/devalibello/vet-clinic/assets/114155239/f8beb94e-378a-4673-a5ea-3b0554513bbd)
I have added a comment to the section of the code you highlighted for clarity.

Thanks 😄